python/dllib/src/bigdl/dllib/nnframes/tree_model.py

```diff
@@ -78,6 +78,7 @@ def __init__(self, jvalue):
         super(XGBClassifierModel, self).__init__()
         invalidInputError(jvalue is not None, "XGBClassifierModel jvalue cannot be None")
         self.value = jvalue
+        self.feature_importances = callZooFunc("dict", "getFeatureImportanceXGBClassifierModel", self.value)
```

Comment:
the first argument should always be float?

Reply:
JMap will be converted to dict in callZooFunc.

Reply:
the first argument of callZooFunc is the single data type, I suppose it can only be float or double and we only use float...
Check other places that calls this function.

Reply:
fixed, change to float.